docs/developer/README.md

```diff
@@ -357,4 +357,7 @@ Then, copy the variables under the section labled `s3`.
 1. On the app, navigate to `\admin`.
 2. Under models, click `Waffle flags`.
 3. Click the `disable_email_sending` record. This should exist by default, if not - create one with that name.
-4. (Important) Set the field `everyone` to `Yes`. This field overrides all other settings 
\ No newline at end of file
+4. (Important) Set the field `everyone` to `Yes`. This field overrides all other settings 
+
+## Request Flow FSM Diagram
+There is a diagram detailing the flow of domain requests and resulting domain objects [here](https://miro.com/app/board/uXjVMuqbLOk=/?moveToWidget=3458764594819017396&cot=14)
```

Comment:
```suggestion

The [.gov Domain Request & Domain Status Digram](https://miro.com/app/board/uXjVMuqbLOk=/?moveToWidget=3458764594819017396&cot=14) visualizes the domain request flow and resulting domain objects.

```

Reply:
Generally when making hyperlinks, you want your linked text to describe what page you're linking to
([plain text language guide on links](https://www.plainlanguage.gov/guidelines/web/write-effective-links/) if you're interested in further reading)

Reply:
Good call out Erin!